src/pages/Login/LoginTemporary.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+
+const LoginTemporary = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState(`DoorWinBell0004@gmail.com`);
+  const [password, setPassword] = useState(`Tmdwhd0711!`);
+
+  const inputEmail = e => {
+    setEmail(e.target.value);
+  };
+
+  const inputPassword = e => {
+    setPassword(e.target.value);
+  };
+
+  const login = () => {
+    fetch('http://10.58.52.191:3000/users/login', {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    })
+      .then(res => res.json())
+      .then(res => localStorage.setItem('TOKEN', res.accessToken))
+      .then(() => navigate('/'))
```

Comment:
```suggestion
      .then(res => res.json())
      .then(res => {
      localStorage.setItem('TOKEN', res.accessToken)
      navigate('/')
      })
```
토큰을 담는것까지는 괜찮은데, 저기서 반환된 값이 없는데 굳이 .then 체이닝을 할 필요가 있을까싶습니다.
setItem 하고 바로 navigate를 호출해주면 됩니다.

Reply:
네 감사합니다!